x-pack/plugins/maps/public/classes/tooltips/es_tooltip_property.ts

```diff
@@ -48,7 +48,12 @@ export class ESTooltipProperty implements ITooltipProperty {
 
     const indexPatternField = this._getIndexPatternField();
     if (!indexPatternField || !this._field.canValueBeFormatted()) {
-      return _.escape(this.getRawValue());
+      const rawValue = this.getRawValue();
+      if (Array.isArray(rawValue)) {
+        return _.escape(rawValue.join());
```

Comment:
@nreese what would be the desired behavior here for array values? Should this join as is, with a `, ` separator,  just show the first value (existing impl), or something else?

Reply:
`this._field.canValueBeFormatted` will only be false for aggregation values that should not use formatters (like count and carnality), so the if statement should never be triggered when the value is an array. I don't think any check is needed for an array type here.

Reply:
Good to know -- thanks! Since `_.escape` only takes a `string` I'll still need to check for an array, or can short circuit in the above conditional if that's preferred.